services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from datetime import datetime
+from db.models import Order, Ticket, User
+from db.models import MovieSession
+
+
+def create_order(tickets: list, username: str, date: str = None) -> User:
```

Comment:
The return type annotation for `create_order` should be `Order`, not `User`, as the function returns an `Order` object.